submission/races.go

```diff
@@ -137,7 +137,7 @@ func (sub *safeSubmissionState) setResult(logURL string, sct *ct.SignedCertifica
 		}
 	}
 
-	// Cancel any pending Log-requests for which there're no more awaiting
+	// Cancel any pending Log-requests for which there are no more awaiting
```

Comment:
I don't think the abbreviation adds any value. There are on the order of 1000x more uses of the expansion than the contraction according to a Google search...